### DISCUSSION TITLE
Ajout par lots des chargés d'orientation dans les carnets existants

### DISCUSSION THREAD

**Discussion by lkouad** [SocialGouv/carnet-de-bord#1322] [Ideas]
#### Problème
Dans le déploiement CD08 nous avons crée le profil "Chargé d'orientation" comme nous disposons de l'ensemble des bénéficiaires du rsa du Département la mise à jour est fastidieuse et se fait au fil de l'eau par les CO. Ce qui implique que les CO n'ont pas une vu exhaustive de leur portefeuille et que les Référents ne voient pas le contact du CO dans le notebook_member du bénéficiaire
#### Solution
Mise à jour des notebook_member en ajoutant par lot les co [cf mattermost](https://github.com/SocialGouv/carnet-de-bord/issues/1099#issue-1378145357) 
- Par critères de sectorisation des CO à l’exception du secteur de Charleville-Mézières code postal 08000 (Fichier csv)
- Par découpage alphabétique du nom de famille du bénéficiaire pour le secteur de Charleville-Mézières  code postal 08000

**Comment by lkouad** [comment]
@lionelB c'est ok pour toi?

**Comment by lionelB** [comment]
Je m'en occupe dans la journée

**Comment by lkouad** [comment]
@lionelB vu la complexité en fin de compte, que faisons nous, peut-être devrions nous renoncer à faire la mise à jour du stock avec les CO non?

**Comment by lionelB** [comment]
Vu l'évolution de ces données, j'ai l'impression qu'un import `oneshot` ne serait pas utile très longtemps malheureusement car il faudrait implémenté une solution qui gère l'affectation aussi pour les nouveau entrants. or cette solution est complexe à mettre en œuvre.

**Comment by lkouad** [comment]
> Vu l'évolution de ces données, j'ai l'impression qu'un import `oneshot` ne serait pas utile très longtemps malheureusement car il faudrait implémenté une solution qui gère l'affectation aussi pour les nouveau entrants. or cette solution est complexe à mettre en œuvre. 

Je suis ok avec toi, je pense qu'il faut refechir a une solution plus simple. Es-tu d'accord pour que je la transforme en discussion et qu'on puisse y réfléchir au prochain réfinement?

**Comment by lionelB** [comment]
yep !
